src/web_server.cpp

```diff
@@ -229,12 +229,7 @@ void buildStatus(DynamicJsonDocument &doc) {
   doc["evse_connected"] = (int)evse.isConnected();
 
   create_rapi_json(doc);
-
-  doc["status"] = evse.getState().toString();
```

Comment:
I think this is still needed

Reply:
oups clearly

Reply:
I remember why I've removed thoses, they are already in create_rapi_json()